sydent/http/servlets/accountservlet.py

```diff
@@ -42,9 +42,8 @@ def render_GET(self, request):
             "user_id": account.userId,
         }
 
-    @jsonwrap
     def render_OPTIONS(self, request):
         send_cors(request)
         request.setResponseCode(200)
-        return {}
+        return ''
```

Comment:
let's try not to introduce *new* python3-incompatibility. You're supposed to return a `bytes` from `render` methods:

```suggestion
        return b''
```

likewise everywhere else

Reply:
ah yep, good point